server/routes/messaging.ts

```diff
@@ -78,14 +104,22 @@ router.route("/api/messaging/coordinator-recruitment-text").post(
   adminProtect,
   asyncHandler(async (req: Request, res: Response) => {
     checkNodeEnvIsProduction(res);
-    await makeRequest(
-      process.env.COORDINATOR_RECRUITMENT_TEXT_WEBHOOK,
-      res,
-      "Unable to start 'Send Coordinator Recruitment Text (Wed afternoon)' Make automation."
+
+    logger.info("[POST] /api/messaging/coordinator-recruitment-text");
+
+    const resp = await sendTextWebhook(
+      process.env.COORDINATOR_RECRUITMENT_TEXT_WEBHOOK!,
+      res.locals.user["Twilio Number"]
     );
-    res.status(OK).json({
-      message:
-        "'Send Coordinator Recruitment Text (Wed afternoon)' Make automation started.",
+
+    const respText = await resp.text();
+
+    const message = resp.ok
+      ? "'Send Coordinator Recruitment Text (Wed afternoon)' Make automation started."
+      : respText;
+
```

Comment:
Let's add the server logging for success/failed here, too, like you have for other ones.

Reply:
Added the logging!